scripts/start/test_runs.R

```diff
@@ -42,27 +42,27 @@ cfg$gms$c_timesteps <- timeSteps
 
 
 # Reference and Policy run for SSP1, SSP2 and SSP5
-for(ssp in c("SSP1","SSP2EU","SSP5")) {
+for(ssp in c("SSP1","SSP2","SSP5")) {
 
   cfg$title <- .title(cfg, paste(ssp,"Ref",sep="-"))
   cfg <- setScenario(cfg,c(ssp,"NPI","rcp7p0"))
   cfg$gms$c56_mute_ghgprices_until <- "y2150"
-  cfg$gms$c56_pollutant_prices <- paste0("R32M46-",ssp,"-NPi")
-  cfg$gms$c60_2ndgen_biodem    <- paste0("R32M46-",ssp,"-NPi")
+  cfg$gms$c56_pollutant_prices <- paste0("R32M46-",ifelse(ssp=="SSP2",paste0(ssp,"EU"),ssp),"-NPi")
```

Comment:
https://rdrr.io/r/base/ifelse.html
> Further note that if(test) yes else no is much more efficient and often much preferable to ifelse(test, yes, no) whenever test is a simple true/false result, i.e., when length(test) == 1. 

```suggestion
  cfg$gms$c56_pollutant_prices <- paste0("R32M46-", if (ssp=="SSP2") "SSP2EU" else ssp,"-NPi")
```

(also applies to ifelse further below)